tests/Model/IndexInfoTest.php

```diff
@@ -166,40 +142,12 @@ public function testIs2dSphere(): void
             'v' => 2,
             'key' => ['pos' => '2dsphere'],
             'name' => 'pos_2dsphere',
-            'ns' => 'foo.bar',
         ]);
 
         $this->assertSame(2, $info->getVersion());
         $this->assertSame(['pos' => '2dsphere'], $info->getKey());
         $this->assertSame('pos_2dsphere', $info->getName());
-        $this->assertSame('foo.bar', $info->getNamespace());
```

Comment:
How did this work previously if `getNamespace()` was already emitting `E_USER_DEPRECATED`?

Reply:
That's an excellent question 🤷‍♂️